src/view/com/composer/Composer.tsx

```diff
@@ -97,6 +99,18 @@ import {TextInput, TextInputRef} from './text-input/TextInput'
 import {ThreadgateBtn} from './threadgate/ThreadgateBtn'
 import {useExternalLinkFetch} from './useExternalLinkFetch'
 import hairlineWidth = StyleSheet.hairlineWidth
+// temp, ensure globalThis.expo is defined. hopefully Expo will fix this upstream
+import {ensureNativeModulesAreInstalled} from 'expo-modules-core/build/ensureNativeModulesAreInstalled'
+
+// stupid method to ensure right order of module execution
+ensureNativeModulesAreInstalled()
+const VideoPreview = lazy(async () => {
+  const {VideoPreview} = await import('./videos/VideoPreview')
+  return {default: VideoPreview}
+})
```

Comment:
ik we were talking about not using `VideoPreview`. if we don't, do we still need the `ensureNativeModulesAreInstalled()`? fine to leave for now in this gated PR but just want to know if we'll have to leave that there.